package.json

```diff
@@ -1,14 +1,17 @@
 {
   "devDependencies": {
     "@vuepress/plugin-google-analytics": "1.8.2",
+    "js-yaml": "^4.1.0",
     "vuepress": "1.8.2",
     "vuepress-plugin-check-md": "0.0.2",
+    "vuepress-plugin-element-tabs": "^0.2.8",
     "vuepress-plugin-sitemap": "2.3.1"
   },
   "scripts": {
     "docs:dev": "vuepress dev docs",
     "docs:build": "vuepress build docs",
-    "docs:check-md": "vuepress check-md"
+    "docs:check-md": "vuepress check-md",
```

Comment:
FYI - @alexfornuto this is a nice way to check that all the links still work / not broken. Highly recommended.

Reply:
Thanks, used in 669b3e2a3d9e04fe017ae47e3c43570dea59307b